uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/request-modal/details/leave.html

```diff
@@ -5,13 +5,13 @@
       <div class="col-xs-8">
         <div class="btn-group btn-group-justified">
           <label
-            ng-disabled="$ctrl.isMode('view') || $ctrl.isRole('manager')"
+            ng-disabled="$ctrl.isMode('view') || ($ctrl.isRole('manager') && !$ctrl.isLeaveType('sickness'))"
```

Comment:
ps refer to PCHR-1101 and it clearly specifies that only for sickness even dates are enabled otherwise not